docs/api-docs/partner/technology-program

```diff
@@ -0,0 +1,132 @@
+
+3. Enter your company email address.
+   
+4. Click the **Technology Partner Program** button.
+   
+![Partner Registration](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/becoming-a-partner-02.png "Partner Registration")
+
+5. Complete the member registration.
+6. Click the **Complete Registration** button. 
+7. Once your application is approved, check your email for your Partner ID.
+
+
+Once approved, you will receive one or more emails listing your partner credentials and next steps. Your **Partner ID** will be required to submit your app for App Marketplace consideration. If you do not receive these credentials by email, please contact [appstore@bigcommerce.com](mailto:appstore@bigcommerce.com).
+
+## App Marketplace Overview
+To build a Marketplace app, you will need:
+
+* A sandbox store
+* A developer account on the [BigCommerce Developer Portal](https://devtools.bigcommerce.com/my/apps)
+* BigCommerce partner credentials 
+
+We encourage you to create your sandbox store and your Dev Tools account using the same login credentials. A sandbox store cannot process transactions and is for developing and testing apps without the 15-day time limit of a trial store. 
+
+> **Note**
+> Your Developer Portal account is an entirely separate account outside of the Partner Portal.
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--warning">
+<div class="HubBlock-content">
+    
+<!-- theme: warning -->
+
+### Trials erased after 15 days
+> Although you do not need a sandbox store for app development, it is best practice. If you opt to develop on a trial store, keep in mind the store and data will be erased after 15 days, and you will have to create a new trial store to proceed.
+
+
+</div>
+</div>
+</div>
+
+### Creating a sandbox store
+
+1. Go to the [BigCommerce Pricing](https://www.bigcommerce.com/essentials/pricing/) page and select **Start your free trial**.
+
+2. When prompted for an email address, enter the _same_ email you used when applying to become a partner.
+
+3. In the trial store setup form, enter a store name, password, and other details. Then select **Create your store**.
+
+4. Log into your Partner account to access the [Partner Portal](https://partners.bigcommerce.com). 
+
+5. Navigate to **Support > Contact Support** and select **Create a Support Case**.
+
+6. While completing the form, use "Partner Sandbox" as the subject line. In the case description, enter your new sandbox store's canonical URL (for example: http://your-store-name.mybigcommerce.com) and request conversion to a sandbox. 
+
+7. Under **Issue Type**, select **Merchant Account Services**. Click **Create Case**. We will flag this new trial store as a sandbox to ensure your store access continues beyond the 15-day trial period. This process can take several days.
+
+8.  Log into your new trial store using the same email associated with your partner login.
+
+### Marketplace apps
```

Comment:
I think this section should be removed from this article and included under Application Development.
``suggestion
### Marketplace apps
```